addons/map/stringtable.xml

```diff
@@ -77,6 +77,12 @@
             <German>Blue Force Tracking</German>
             <Czech>Blue Force Tracking</Czech>
         </Key>
+        <Key ID="STR_ACE_Map_BFT_Enabled_DisplayName">
+            <English>Enable</English>
```

Comment:
Minor thing, but I would change this to `BFT Enabled` to make it easier to find on the server settings dialog.  I think it should still fit on the module.